### PR TITLE
Fill in unknown for stackdriver metric empty labels

### PR DESCRIPTION
**Pull request by bianpengyuan**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: extensions/stackdriver/metric/record.cc [hunk 1]
```diff
@@ -93,38 +95,45 @@ TagKeyValueList getOutboundTagMap(
     const ::Wasm::Common::FlatNode& peer_node_info,
     const ::Wasm::Common::RequestInfo& request_info) {
   TagKeyValueList outboundMap = {
-      {meshUIDKey(), flatbuffers::GetString(local_node_info.mesh_id())},
-      {requestProtocolKey(), std::string(request_info.request_protocol)},
+      {meshUIDKey(),
+       unknownIfEmpty(flatbuffers::GetString(local_node_info.mesh_id()))},
+      {requestProtocolKey(),
+       unknownIfEmpty(std::string(request_info.request_protocol))},
```
Comment (gargnupur) [inline extensions/stackdriver/metric/record.cc:101]: for things we get directly from request_info.. shouldn't we set them to "unknown" if it's empty for all telemetry and not just for metrics?

Comment (bianpengyuan) [reply extensions/stackdriver/metric/record.cc]: It is going to make the code in context quite verbose. All wasm ABI/API set strings to empty if value is not found. If we want to set unknown in context, that would mean each getValue call would need to be wrapped by an `if (value.empty) then unknown` block. I think it is fine to make code in context follow wasm ABI/API convention and let each caller decide what to do with empty values.

Comment (gargnupur) [reply extensions/stackdriver/metric/record.cc]: Sounds good!

**Pull request merged**